setup.py

```diff
@@ -7,7 +7,7 @@
         'tensorflow-probability~=0.10.0',
     ],
     'torch': ['torch~=1.2'],
-    'jax': ['jax~=0.1,>0.1.51', 'jaxlib~=0.1,>0.1.33'],
+    'jax': ['jax~=0.2.4', 'jaxlib~=0.1.56'],
```

Comment:
I think we should pin down to the minor version here as it seems that JAX is still new enough that there are breaking API changes on minor releases (which is totally fair given that it is a `v0.X` library still).